pkg/api/errors/errors.go

```diff
@@ -226,7 +226,7 @@ func NewServerTimeout(kind, operation string, retryAfterSeconds int) error {
 		Details: &unversioned.StatusDetails{
 			Kind:              kind,
 			Name:              operation,
-			RetryAfterSeconds: retryAfterSeconds,
+			RetryAfterSeconds: int32(retryAfterSeconds),
```

Comment:
I don't expect it makes a difference for this field, but casting like this is not safe in general as it potentially hides a cross-platform bug.